### PR TITLE
Include scope in canvas auth request

### PR DESCRIPTION
**Pull request by hmstepanek**
This is a partial fix for https://github.com/hypothesis/lms/issues/806.

### CODE REVIEW

**Comment by seanh** [comment]
This looks good to me and seems to work locally. The only thing I want to think about is whether we should add any more scopes. You suggested a couple (https://github.com/hypothesis/lms/issues/806#issuecomment-516091760). I want to have a look over them...

**Comment by hmstepanek** [comment]
Yes, I can do that. I think merging this with just these two scopes is good for now as well since as you said it’s hard to anticipate what scopes we will need in the future.

File: lms/views/api/canvas/authorize.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def authorize(self):
                         "response_type": "code",
                         "redirect_uri": self.request.route_url("canvas_oauth_callback"),
                         "state": CanvasOAuthCallbackSchema(self.request).state_param(),
+                        "scope": "url:GET|/api/v1/courses/:course_id/files url:GET|/api/v1/files/:id/public_url",
```
Comment (robertknight) [inline lms/views/api/canvas/authorize.py]: If we end up with many scopes in future I think it would be good to note what the various scopes are for.

**Comment by robertknight** [comment]
> I think merging this with just these two scopes is good for now as well since as you said it’s hard to anticipate what scopes we will need in the future.

Yes, I mainly wanted to flag up that when we do add new scopes, we'll need to consider existing scoped dev keys that exist.

In this first PR, we're not adding any new scopes since all authorizations currently implicitly require all scopes.

**Pull request merged**